quality_cuts/new_var_dist.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from tree_importer import tree_importer
+import gc
+
+"""
+ All the nonesences ( $\chi$2 < 0), inf and nan were deleted. Also applied
+ quality cuts based on detector geometry. Full description could be found in
+ https://docs.google.com/document/d/11f0ZKPW8ftTVhTxeWiog1g6qdsGgN1mlIE3vd5FHLbc/edit?usp=sharing
+
+"""
+
+from concurrent.futures import ThreadPoolExecutor
+executor = ThreadPoolExecutor(8)
+
+
+
+file = tree_importer('/home/olha/CBM/dataset/10k_events_PFSimplePlainTree.root',
+                     'PlainTree')
+
+gc.collect()
+
+
+df_original= pd.DataFrame(data=file)
+del file
+
+
+new_labels= ['chi2geo', 'chi2primneg', 'chi2primpos', 'chi2topo', 'cosineneg',
+       'cosinepos', 'cosinetopo', 'distance', 'eta', 'l', 'ldl',
+       'mass', 'p', 'pT', 'phi', 'px', 'py', 'pz', 'rapidity',
+             'x', 'y', 'z', 'daughter1id', 'daughter2id', 'isfrompv', 'pid', 'issignal']
+
+
+df_original.columns = new_labels
+
+
+sgnal = df_original[df_original['issignal']==1]
+
+
+bg = df_original[df_original['issignal']==0]
+
+
```

Comment:
make a function and save it in a separate python file as quality_cuts. The other cuts will be named as pre-selection cuts